doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -125,6 +125,9 @@ Bluetooth samples
 * Updated some samples with support for :ref:`zephyr:thingy53_nrf5340` in non-secure configuration.
 * :ref:`ble_llpm` sample - Added role selection.
   The user now selects the role for each board by typing "m" or "s" in the terminal emulator.
+* Changed the default sample for the :ref:`ble_rpc` to the :ref:`peripheral_uart` sample. This sample runs out of the box with a serialized Bluetooth Low Energy Host.
+* Added the :ref:`ble_rpc_host:` child image configuration overlay for the :ref:`peripheral_hids_mouse` sample as an example how to configure application running a serialized Bluetooth Low Energy Host.
+  The :ref:`peripheral_hids_mouse` runs out the box with the :ref:`ble_rpc` library.
```

Comment:
```suggestion
* :ref:`peripheral_uart` sample is now the default sample for the :ref:`ble_rpc` library.
  The sample runs out of the box with a serialized Bluetooth Low Energy Host.
* :ref:`peripheral_hids_mouse` sample now comes with the :ref:`ble_rpc_host` child image configuration overlay.
  The overlay shows how to configure application running a serialized Bluetooth Low Energy Host.
  The :ref:`peripheral_hids_mouse` runs out the box with the :ref:`ble_rpc` library.
```

Reply:
The overlay shows how to configure **an** application...

Reply:
Done